commit 086e6fdce887dd68e51b7bac4a2f21cea9a4fe01
Author: Steve Dickson <steved@redhat.com>
Date:   Fri Sep 4 14:15:53 2020 -0400

    rpc.idmapd: Do not free config variables
    
    Commit 93e8f092e added a conf_cleanup() call to clean
    up memory after the config file was parsed. It turns
    out that memory still needed and it is not very much
    so the call is removed.
    
    Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1873965
    
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/utils/idmapd/idmapd.c b/utils/idmapd/idmapd.c
index f3d2314..51c71fb 100644
--- a/utils/idmapd/idmapd.c
+++ b/utils/idmapd/idmapd.c
@@ -306,9 +306,6 @@ main(int argc, char **argv)
 			serverstart = 0;
 	}
 
-	/* Config memory is no longer needed */
-	conf_cleanup();
-
 	while ((opt = getopt(argc, argv, GETOPTSTR)) != -1)
 		switch (opt) {
 		case 'v':
